grpc-protocol/src/main/java/com/linecorp/armeria/internal/common/grpc/protocol/UnaryGrpcSerializationFormats.java

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright 2021 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package com.linecorp.armeria.internal.common.grpc.protocol;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+import com.linecorp.armeria.common.SerializationFormat;
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+/**
+ * {@link SerializationFormat} instances supported by unary gRPC service and client that do not
+ * depend on gRPC stubs.
+ */
+@UnstableApi
+public final class UnaryGrpcSerializationFormats {
```

Comment:
We usually don't add `@UnstableApi` for internal APIs.
I think users may use `UnaryGrpcSerializationFormats` when sending a request.
If so, isn't it better to put `UnaryGrpcSerializationFormats` in the `common.grpc.protocol` instead of `internal.common.grpc.protocol`?

Reply:
That's where it was initially, but @trustin asked to move it here, see rationale in https://github.com/line/armeria/pull/3716#discussion_r675671534. Should I remove `@UnstableApi`, or move `UnaryGrpcSerializationFormats` back to `common.grpc.protocol`?

Reply:
We had a discussion about this earlier in this PR actually :laughing: and I suggested to moving it to the current location. https://github.com/line/armeria/pull/3716#discussion_r675671534

Reply:
Sorry I missed it. 😅